app/Jobs/IndexWallets.php

```diff
@@ -17,6 +17,6 @@ public function handle(): void
 
     protected function elementsToIndexQuery(int $latestIndexedTimestamp): Builder
     {
-        return Wallet::getSearchableQuery()->where(DB::raw('(SELECT MIN(transactions.timestamp) FROM transactions WHERE transactions.recipient_id = wallets.address)'), '>', $latestIndexedTimestamp);
+        return Wallet::getSearchableQuery()->where(DB::raw('(SELECT MAX(transactions.timestamp) FROM transactions WHERE transactions.recipient_id = wallets.address)'), '>', $latestIndexedTimestamp);
```

Comment:
With this change any wallet that has a transaction associated that was added before the last time we checked is considered as one that needs to be reindexed

Reply:
won't this only update wallets that received a transaction while the one sending it will not be updated? For example wallet A has 10 ARK and sends 2 ARK to wallet B who had 5 ARK at the time, so B is updated as to have 7 ARK as it's the recipient while wallet A remains at 10 ARK in the index while it actually has 8 now?

Reply:
Alex noted in slack that also the multipayments are ignored as the sender === recipient for that transaction type. Instead it has a special asset that contains the list of recipients that would need to be checked

Reply:
@ItsANameToo yes, and yes, will see if i can update the queries to consider multipayments and senders